templates/includes/sidebar.html

```diff
@@ -1,6 +1,20 @@
 <ul class="sidebar-nav">
-  <li><a href="{% url "taxi:index" %}">Home page</a></li>
-  <li><a href="{% url "taxi:manufacturer-list" %}">Manufacturers</a></li>
-  <li><a href="{% url "taxi:car-list" %}">Cars</a></li>
-  <li><a href="{% url "taxi:driver-list" %}">Drivers</a></li>
+  {% if user.is_authenticated %}
+    <li>
+      User: <a href="{% url 'taxi:driver-detail' user.id %}">{{ user.username }}</a>
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.